src/Frontend/Modules/Slideshows/Widgets/Detail.php

```diff
@@ -50,4 +50,16 @@ private function parse()
         // assign data
         $this->tpl->assign('slideshow', $this->item);
     }
+
+    /**
+     * @return string
+     */
+    private function getSlideShowTemplate()
+    {
+        if (Theme::getTheme() != 'code') {
```

Comment:
also, wouldn't it be a good idea to check if this template still actually exists?